src/libsync/account.cpp

```diff
@@ -160,6 +162,12 @@ void Account::setCredentials(AbstractCredentials *cred)
 
     _am = _credentials->createAM();
 
+    const QString cacheLocation = QStringLiteral("%1/accounts/%2/network/").arg(QStandardPaths::writableLocation(QStandardPaths::CacheLocation), _uuid.toString());
+    qCDebug(lcAccount) << "Cache location for account" << this << "set to" << cacheLocation;
+    _networkCache->setCacheDirectory(cacheLocation);
+    _networkCache->setMaximumCacheSize(10 * 1024 * 1024);
```

Comment:
can you use our literals?

Reply:
I want to get rid of our custom setting completely. We added it yesterday just for testing.